README.md

```diff
@@ -23,55 +23,12 @@ yarn
 3. Build the code and run the local version of the UI
 
 ```sh
-yarn start
+yarn playground
 ```
 
-This command will run a very simple HTML file defined in [./index.html](./index.html).
-Any changes made to the [./index.html](./index.html) or [./src](./src/) will cause the page to auto-reload
-without needing to restart the server.
+## Playground
 
-### Playground View
+The default consent interface is built with flexibility to choose between different "view states". These can be viewed in the [Consent Manager UI playground here](https://consent-manager-ui.vercel.app/src/playground).
 
-![The Consent UI development playground](https://user-images.githubusercontent.com/7354176/187821040-5a4617fa-8c50-43c5-80af-5b4d18b344b4.png)
+<img width="1468" alt="Screen Shot 2022-12-14 at 2 38 05 AM" src="https://user-images.githubusercontent.com/7354176/207547449-3519436f-2f2d-487f-b9d6-af8f803ff71f.png">
 
-## View States
```

Comment:
hm i dont think we should remove these, we link to this from the admin dashboard, and i imagine folks have linked directly to each of these view states